CRUD/AddCart/etc/module.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Module/etc/module.xsd">
+    <module name="CRUD_AddCart" setup_version="1.0.0">
```

Comment:
@msakila532 If you use declarative schema we don't have to specify setup_version.